c++/src/kj/filesystem-disk-win32.c++

```diff
@@ -31,6 +31,10 @@
 
 // Request Vista-level APIs.
 #define WINVER 0x0600
+
+#ifdef _WIN32_WINNT
```

Comment:
Hmm, maybe the #define _WIN32_WINNT should be moved above all the other includes, to make sure no system header defines it first?

Reply:
Sure... I was just making the change that would preserve existing behavior, but doing the define earlier might prevent other conflicts.  I wasn't sure if something else was unconditionally defining the value, but it looks like the conflicting definition was in _mingw.h, which seems compatible with moving the definition earlier:

https://github.com/Alexpux/mingw-w64/blob/master/mingw-w64-headers/crt/_mingw.h.in#L225